src/main/resources/config/config.yaml

```diff
@@ -56,7 +56,8 @@ rateLimiter:  # rate = noOfReq per perMillis
 redis:
   endpoint: ${REDIS_URL:-localhost:6379}
   ssl: false
-  timeout: 2s
+  commandTimeout: ${REDIS_COMMAND_TIMEOUT:-250ms}
```

Comment:
In the PR can you please link to the perf test runs that demonstrate this doesn't invoke unnecessary command timeouts whilst Redis is running as expected please?

Reply:
Updated.